### ISSUE TITLE
How were im2latex_test.lst, im2latex_train.lst, and im2latex_validate.lst generated?

### ISSUE THREAD

**Issue by mitar** [Miffyli/im2latex-dataset#6]
In the [dataset](https://zenodo.org/record/56198#.WNJclnV94jA) there are `im2latex_test.lst`, `im2latex_train.lst`, and `im2latex_validate.lst` files. How were those generated?

**Comment by Miffyli** [comment]
I used [scikit-learn's train_test_split](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) method. I can not recall the exact ratios but I believe I first split whole data into train (80%) and test (20%). Then I split the test set into evaluation (50% of test) and validation (50%) sets.
